src/main/java/seedu/address/model/event/Event.java

```diff
@@ -75,7 +75,6 @@ public boolean isSameEvent(Event otherEvent) {
 
         return otherEvent != null
                 && otherEvent.getName().equals(getName())
-                && otherEvent.getDescription().equals(getDescription())
                 && otherEvent.getLocation().equals(getLocation())
                 && otherEvent.getEventDate().equals(getEventDate());
```

Comment:
One thing to consider, when comparing `name` and `location`, should we use `equalsIgnoreCase`?

Reply:
Good point - fixed.